VimeoNetworking/VimeoNetworking/Models/VIMSeason.m

```diff
@@ -85,4 +85,18 @@ - (void)parseInteractions
     self.interactions = interactions;
 }
 
+- (BOOL)containsDRMProtectedContent
+{
+    VIMInteraction *buyInteraction = [self interactionWithName:VIMInteractionNameBuy];
```

Comment:
Will relying on `isForDRMProtectedContent` work in both the iOS and tvOS apps given that they are pointed at different API versions? 3.3 and 3.3.1